### DISCUSSION TITLE
Add annotations

### DISCUSSION THREAD

**Discussion by Assitan** [react-syntax-highlighter/react-syntax-highlighter#412] [General]
Hello,

How can I add annotations ? For example, I'd like to select a line and a tooltip appears with some text.

Thanks

**Comment by Assitan** [comment]
I found a way !

**Reply by Henzelix** [reply]
Can you explain how you did it?

**Reply by Assitan** [reply]
I used "lineProps" by creating a function that add a class and an id to the selected line. And I get these id / class in a div (the annotation).

![Screenshot from 2021-08-12 14-08-49](https://user-images.githubusercontent.com/3190142/129195358-c0d49113-19e4-4f89-8b82-13089f7dc3bd.png)
![Screenshot from 2021-08-12 14-17-42](https://user-images.githubusercontent.com/3190142/129195368-32ac0523-60a4-4f05-8b66-bf22b12aaf8a.png)
